### ISSUE TITLE
How the result.txt format looks like

### ISSUE THREAD

**Issue by LinHungShi** [asharakeh/kitti_native_evaluation#2]
Hi, When I copy the ground truth label to my prediction directory and run evaluation file, if I am right, AP is supposed to be 1 since the ground truth and prediction are the same. However, the final metric is pretty low. I am pondering if the prediction format is different from the ground truth label, but haven't figured out the solution. Could you give me some hints? Thanks

**Comment by N-McA** [comment]
This repo is not very useful without the result format specification! I'm happy to help write up a guide if someone can provide pointers on how it works.

**Comment by N-McA** [comment]
So, first discovery is that you need to include a confidence in order for the predictions to be considered. Hence the (very smart) idea of copying the gt as a test does not work. If you append a float to each line however, it does...

**Comment by asharakeh** [comment]
Sorry for the long response time. @N-McA is correct. You need a score for the algorithm to work, as a score threshold is used for constructing the PR curve.

**Comment by langelesl** [comment]
Hi,

I have similar questions like @LinHungShi :

1.To figure out how the evaluation function works, I made the folders following your instruction in README.md, and I copied 5 ground truth files (000000~000004.txt) from label_2 folder of KITTI object Benchmark into the kitti_native_evaluation/results folder as results, and I add a fake confidence (like 0.99) for every detected object at the end of every line in .txt. I hope it could output PR-curve like results in KITTI Object Detection Benchmark. However, after I ran the command "./evaluate_object_3d_offline  data/object/label_2  results", the output in terminal is like this:

![Screenshot from 2019-03-09 22-19-56](https://user-images.githubusercontent.com/45672268/54077531-0a8eed00-42ba-11e9-812d-1bcfdbb0a087.png)

So I would like to ask you some advices about how I should run your evaluation code correctly, and what the evaluated results should look like.

2. I have trained a model to do the 2d object detection on kitti dataset. Is that also possible for your evaluation code to evaluate the results of 2d object detection? If positive, what should I notice in making results.txt?

Thanks again.
